bin/collector

```diff
@@ -3,30 +3,29 @@
 
 Process.setproctitle("discourse prometheus-collector")
 
-[
-  "webrick-#{ENV["WEBRICK_VERSION"]}",
```

Comment:
Previously we would require the gem from the `gems` folder of the plugin.

Reply:
Do we want to keep the `.each` loop for just one element?